index.html

```diff
@@ -129,46 +129,57 @@ <h2>
                     <tr>
                         <th>Time</th>
                         <th>Schedule</th>
+                        <th>Location</th>
                     </tr>
                     <tr>
                         <td>9:15 AM</td>
                         <td>Registration and Morning Coffee</td>
+                        <td>Activities & Recreation Center (ARC)</td>
                     </tr>
                     <tr>
                         <td>10:00 AM</td>
```

Comment:
we could add an entry for "Start Hacking!" that starts at 10:20 in the ARC. other than that, I think everything is in the NCEC